rosidl_generator_py/msg/Strings.msg

```diff
@@ -1,5 +1,9 @@
 string empty_string
 string def_string "Hello world!"
+string def_string_delimiter 'Hello"world!'
+string def_string_delimiter2 "Hello'world!"
+string def_string_delimiter3 'Hello'world!'
+string def_string_delimiter4 "Hello"world!"
```

Comment:
The last two lines don't look like a valid quoted string to me. How can the quotes used on the left and right of the string appear unescaped in the middle?